src/poliastro/maneuver.py

```diff
@@ -202,52 +133,29 @@ def bielliptic(cls, orbit_i, r_b, r_f):
         else:
             t_pericenter = 0 * u.s
 
-        # Initial orbit data
         k = orbit_i.attractor.k
-        r_i = orbit_i.r
-        v_i = orbit_i.v
-        h_i = norm(cross(r_i.to(u.m).value, v_i.to(u.m / u.s).value) * u.m ** 2 / u.s)
-        p_i = h_i ** 2 / k.to(u.m ** 3 / u.s ** 2)
-
-        # Bielliptic is defined always from the PQW frame, since it is the
-        # natural plane of the orbit
-        r_i, v_i = rv_pqw(
-            k.to(u.m ** 3 / u.s ** 2).value,
-            p_i.to(u.m).value,
-            orbit_i.ecc.value,
-            orbit_i.nu.to(u.rad).value,
-        )
+        _, ecc, inc, raan, argp, nu = orbit_i.classical()
```

Comment:
Where are the rest of the variables used? Can't see it after a quick pass

Reply:
So sorry for that @astrojuanlu and thanks for pointing out. I have changed it! They shouldn't be there since those variables are already created in `core`.

Thanks